File: src/main/Python/boj_step8/boj11005.py
import sys
cin = sys.stdin.readline

N, B = map(int, cin().split())
tmp = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ'
result = ''
while N > 0:
    result += tmp[N % B]
    N //= B
print(result[::-1])

File: src/main/Cpp/boj_step8/boj11005.cpp
#include <iostream>
using namespace std;

int main() {
    string digits = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ", result;
    int N, B;
    cin >> N >> B;

    while (N > 0) {
        result = digits[N % B] + result;
        N /= B;
    }

    cout << result;
}